process/consensus.md

```diff
@@ -58,8 +60,8 @@ online, either on GitHub repositories under the WebAssembly organization or in
 official video calls. In the latter case, the consensus vote must be added to
 the agenda at least 24 hours before the video call is scheduled to begin, except
 in the case of general interest votes moving pre-proposals to phase 1, which can
-be added as the discussion proceeds. Consensus decisions are recorded in meeting
-notes and published just like in-person meeting notes are published.
+be added as the discussion proceeds. Consensus decisions are recorded in the
```

Comment:
The paragraph above includes references to video call meetings, (as opposed to "in-person meetings" as the preceding "consensus at meetings" section used to talk about). But now video calls are added to the preceding section as official meetings.
So I think this paragraph should now say something like the following:

"It is critical that work progresses between full-CG meetings: agreed-upon
designs need to move forward, and new ideas need to reach some level of maturity
before being discussed in a full meeting. To that end, this group can reach consensus
online, either on GitHub repositories, or in smaller groups."

That leaves out the part about adding votes to the agenda 24 hours in advance. I guess that should move to the "consensus at meetings" section above.

Reply:
Maybe we should even rename this section, since now it covers Github and small groups, but not VC meetings?